salt/config/__init__.py

```diff
@@ -395,9 +395,9 @@
     'return_retry_timer': int,
     'return_retry_timer_max': int,
 
-    # Specify a returner in which all events will be sent to. Requires that the returner in question
-    # have an event_return(event) function!
-    'event_return': str,
+    # Specify one or more returners in which all events will be sent to. Requires that the returners
+    # in question have an event_return(event) function!
+    'event_return': list,
```

Comment:
OK, it's possible that the warning logic has been changed since I last checked, that comment was from my memory of how it worked, having been the one that wrote that logic.